### ISSUE TITLE
Signature error when attempting to use STS

### ISSUE THREAD

**Issue by mrmcc3** [cognitect-labs/aws-api#27]
Hello, I'm getting the following signature error 
```clojure
{:ErrorResponse {:Error {:Type "Sender",
                         :Code "SignatureDoesNotMatch",
                         :Message "Credential should be scoped to a valid region, not 'ap-southeast-2'. "}},
 :ErrorResponseAttrs {:xmlns "https://sts.amazonaws.com/doc/2011-06-15/"},
 :cognitect.anomalies/category :cognitect.anomalies/forbidden}
```
When calling
```clojure
(aws/invoke sts {:op :AssumeRole :request {:RoleArn arn :RoleSessionName "REPL"}})
```
For reference other APIs work fine 
```clojure
(aws/invoke ec2 {:op :DescribeInstances})
```
Deps
```clojure
{:deps {org.clojure/clojure         {:mvn/version "1.10.0"}
        com.cognitect.aws/api       {:mvn/version "0.8.158"}
        com.cognitect.aws/endpoints {:mvn/version "1.1.11.467"}
        com.cognitect.aws/sts       {:mvn/version "669.2.364.0"}
        com.cognitect.aws/ec2       {:mvn/version "681.2.373.0"}}}
```

**Comment by dchelimsky** [comment]
Per https://docs.aws.amazon.com/general/latest/gr/sigv4_elements.html#sigv4_elements_endpoint, "If you use a service like IAM that uses a globally unique endpoint, use the default region (us-east-1), as part of the Credential authentication parameter (described later in this topic)."

STS is such a service, and implementing ^^ fixes the issue. Fix coming shortly. Release in the next day or so.

**Comment by dchelimsky** [comment]
Released in 0.8.166.
